### DISCUSSION TITLE
Password for admin login???

### DISCUSSION THREAD

**Discussion by rupertcachandler** [deeztek/Hermes-Secure-Email-Gateway#15] [Support] [answered]
HI,

have done clean installs twice and have a complete note of all passwords set.

Can't login to the 9080/admin/logon.cfm interface.

using admin

and then password I set for web administrator, but no luck..

is admin correct?

This is not the VM, but a metal install on a new Ubuntu 18.04 machine.

Rup

**Comment by deeztek** [comment, marked as answer]
Hi,

The default credentials for the Web GUI are:

admin
ChangeMe2!

They are not set during setup. The passwords you set during setup are for the databases. Take a look at the Getting Started guide here:

[https://docs.deeztek.com/books/hermes-seg-administrator-guide/page/getting-started](https://docs.deeztek.com/books/hermes-seg-administrator-guide/page/getting-started)

Please let us know if any more problems.

**Reply by rupertcachandler** [reply]
Hi,

Did try this combination with my first install and no success, maybe the install wasn't successful..

Thanks for your help!!

Rup

**Reply by deeztek** [reply]
Instead of re-installing try resetting the password using the documentation here:

[https://docs.deeztek.com/books/hermes-seg-general-documentation/page/reset-lostforgotten-hermes-seg-admin-password](https://docs.deeztek.com/books/hermes-seg-general-documentation/page/reset-lostforgotten-hermes-seg-admin-password)

**Reply by deeztek** [reply]
If the Web GUI comes up, then the installation was most likely successful. You can can always post the installation log here so that we can take a look.

**Comment by rupertcachandler** [comment]
Cool, thanks!!


I did google for a decent amount of time to see if I could reset the password, couldn’t find that document.


All running now!!


Just need to setup to run as an incoming mail gateway.


Rup
  
  
  


Rupert Chandler 
Tel: +44(0)7968191387

***@***.***

**Reply by deeztek** [reply]
Good to hear! Let us know if you need more assistance.

Cheers!
